frontend/app-development/layout/SettingsModalButton/SettingsModal/components/Tabs/SetupTab/SetupTabContent/SetupTabContent.module.css

```
@@ -0,0 +1,18 @@
+.dateWrapper {
+  display: flex;
+  flex-direction: column;
+  width: 70%;
```

Burde vi heller sette en max-width, slik at dette skalerer bedre mellom skjermstørrelser? 🤔 
```suggestion
  max-width: 470px;
  width: 100%;
```

---

Yes 😄